src/app/@details/[login]/not-found.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
```

Comment:
where is this component used ? i thought it was for contributor modal but finally not isn't ?